import.py

```diff
@@ -171,6 +185,35 @@ def processShapefile(f, stem, inputDir, outputDir, srs, keyField):
 
 
 
+def processRaster(f, stem, dataDir, reprojectedDir, SRIDNamespace, desiredSRID):
+  originalPath = os.path.join(dataDir, f)
+  original = GDALRaster(originalPath, write=False)
+  reprojectedPath = os.path.join(reprojectedDir, f)
+  if str(original.srs.srid) == desiredSRID:
+    print("Skipping reprojection because this file is already in " + SRIDNamespace + ":" + desiredSRID + ".")
```

Comment:
For the shapefiles, I think they always get copied to the reprojected folder, even if they are already in the right SRID. It would be better to be consistent with that behavior, and if it's already in the right srid, just copy it over to the reprojected folder.

Reply:
Oh, this is a good point.  Also since I wrote that code, I added datatype conversion to the reprojection, and the whole reproject+convert is fast enough that we might as well always do it even if it's not necessary.